CampMinder.gemspec

```diff
@@ -20,11 +20,8 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency "bundler"
   spec.add_development_dependency "dotenv", "~> 1.0"
-  spec.add_development_dependency "guard"
-  spec.add_development_dependency "guard-rspec"
-  spec.add_development_dependency "pry", "~> 0.10"
   spec.add_development_dependency "rack-test"
-  spec.add_development_dependency "rails", "~> 6.0.3"
+  spec.add_development_dependency "rails", "6.0.3.7"
```

Comment:
This is to get around mimemagic (do you remember when that library went supernova and took down the ruby community?)